corehq/toggles.py

```diff
@@ -1810,3 +1810,10 @@ def _enable_bi_integration_preview(domain, enabled):
     TAG_PRODUCT,
     [NAMESPACE_DOMAIN],
 )
+
+SESSION_MIDDLEWARE_LOGGING = StaticToggle(
```

Comment:
what about making this a random toggle based on request hash or something so we can control the impact?

Reply:
We could do that latter depending on the findings from a single domain first.

Reply:
I'd assumed this would be done on ICDS - how are you planning to test it?

Reply:
I was intending to do it on any of the ICDS test domains for starters and then perhaps random toggle with very low probability on ICDS, so that logs are not heavy. (user based toggle can't work here since user gets set after session and auth middleware are processed)